src/modules/qt/Makefile

```diff
@@ -16,7 +16,8 @@ CPPOBJS = common.o \
 	kdenlivetitle_wrapper.o \
 	producer_qtext.o \
 	transition_qtblend.o \
-	filter_qtblend.o
+	filter_qtblend.o \
+	filter_qresize.o
```

Comment:
I would suggest the name change to "qtresize" for consistency. I regret that I did not name "qtext" "qttext". I thought I was being clever by reusing the "t" at the time.